src/PhpWord/Writer/HTML/Element/Footnote.php

```diff
@@ -46,7 +46,7 @@ public function write()
 
         $noteId = count($parentWriter->getNotes()) + 1;
         $noteMark = $this->noteType . '-' . $this->element->getRelationId();
-        $content = "<a name=\"{$noteMark}\"><a href=\"#note-{$noteId}\" class=\"NoteRef\"><sup>{$noteId}</sup></a>";
+        $content = "<a name=\"{$noteMark}\"><a href=\"#note-{$noteId}\" class=\"NoteRef\"><sup>{$noteId}</sup></a></a>";
```

Comment:
The HTML5 standard does not allow nested <a> tags.

It needs to be rewritten like this
```php
$content = "<a name=\"{$noteMark}\"></a><a href=\"#note-{$noteId}\" class=\"NoteRef\"><sup>{$noteId}</sup></a>";
```